grunt/index.d.ts

```diff
@@ -796,8 +796,8 @@ declare namespace grunt {
              *
              * @note taskFunction.apply(scope: grunt.task.IMultiTask<any>, args: any[])
              */
-            registerMultiTask(taskName: string, taskFunction: Function): void
-            registerMultiTask(taskName: string, taskDescription: string, taskFunction: Function): void
+            registerMultiTask(taskName: string, taskFunction: (this: IMultiTask<any>, ...args: any[]) => void): void;
+            registerMultiTask<T extends any>(taskName: string, taskDescription: string, taskFunction: (this: IMultiTask<T>, ...args: any[]) => void): void;
```

Comment:
`T extends any` defaults to `any` instead of `{}`? If I want to explicitly provide a type I can do that with `registerMultiTask<T>(...)`.

I missed to do the same on the first overload